netbox/extras/tables.py

```diff
@@ -171,6 +173,7 @@ class TaggedItemTable(BaseTable):
     class Meta(BaseTable.Meta):
         model = TaggedItem
         fields = ('content_type', 'content_object')
+        exclude = ('id', )
```

Comment:
Alternatively, you can create a custom field (or use a template field) and link to the appropriate objects view

Reply:
Actually, I said that wrong. TaggedItem does have an ID for the TaggedItem object itself, which is not the same as the corresponding object's ID. The TaggedItem ID isn't useful information to expose, as I don't think that's accessible/usable anywhere?

I could make the ID column expose the object ID instead, but I wasn't sure how useful that is for a couple of reasons. 

1. this table isn't exportable, which is the primary reason this change was requested in the first place (though, the ID column is currently on plenty of other non-exportable table views, so that's kind of moot).
2. object ID isn't inherently unique in this table, so needs to be used in conjunction with the type column in order to extract useful information
3. If linkify-ing it, the object column is already linked to that object's detail page

Though, now that I actually look at it again, it's a lot easier to properly include the object ID and linkify it than I initially thought it would be, so this isn't really a big deal to add/get working after all.